tools/msgenc.py

```diff
@@ -51,21 +52,27 @@ def cvt_str(m):
 def main():
     parser = argparse.ArgumentParser(description="Encode message_data_static text headers")
     parser.add_argument("charmap", help="path to charmap file specifying custom encoding elements")
-    parser.add_argument("input", help="path to file to be encoded")
-    parser.add_argument("output", help="encoded file")
+    parser.add_argument("--input", help="path to file to be encoded, or - for stdin", default="-")
```

Comment:
FWIW I'd rather this be positional and charmap to be --charmap than the other way around

Also `-o` for `--output` might be nice, but not important since I don't think too many people will run this by hand

Reply:
lmk if this is better